spec/rubocop/cop/style/redundant_parentheses_spec.rb

```diff
@@ -723,6 +723,16 @@ def foo
     expect_no_offenses('x = *(y || z)')
   end
 
+  it 'accepts parentheses around `case` operator in splat' do
```

Comment:
Shouldn't it be "case expression" rather than "case operator"?

```suggestion
  it 'accepts parentheses around `case` expression in splat' do
```

Reply:
Agreed, fixed